README.md

```diff
@@ -93,7 +92,7 @@ per platform, and can also be configured for cross-compilation.
 System-wide installation of the resulting sentry library is also possible via
 CMake.
 
-Building the Crashpad Backend requires a `C++14` compatible compiler.
+Building the Crashpad Backend requires a `C++20` compatible compiler.
```

Comment:
@supervacuus you mentioned linking this to the tracking ticket, but not sure which one you were referring to

Reply:
https://github.com/getsentry/sentry-native/issues/1041